components/common/EventPopup/EventPopup.tsx

```diff
@@ -1,37 +1,41 @@
-import React from 'react'
-import { Dialog } from '@mui/material';
+import React from "react";
+import { Dialog } from "@mui/material";
 import styles from "../../../styles/components/EventPopup.module.scss";
-import { Josefin_Sans } from 'next/font/google';
-import Image from 'next/image';
-import {RxCross2} from 'react-icons/rx';
-import { Common } from '../../../constants/common';
+import Image from "next/image";
+import { RxCross2 } from "react-icons/rx";
+import { Common } from "../../../constants/common";
 
 type Props = {
     title: string;
-    subTitle: string;
-    description: string[];
+    subTitle?: string;
+    description?: string;
     imageUrl: string;
     startDate: Date;
     endDate: Date;
+    venue?: string;
     handleClose: any;
-}
```

Comment:
please pass id: number and ended: boolean as a prop as well, I need this for calling POST `/v1/events/{id}/end`